feature-flags/feature-flags.development.ts

```diff
@@ -6,11 +6,8 @@ export const flagsDevelopment: FeatureFlagsInterface = {
   FEATURE_TRANSPARENCY_NEW_TABLE: true,
   FEATURE_CU_INDEX_NEW_TABLE: true,
   FEATURE_CU_ABOUT_NEW_CONTAINER: true,
```

Comment:
This is no longer used so it can be safely deleted

Reply:
done